### PR TITLE
Minor updates

### PR DESCRIPTION
**Pull request by funkytaco**
Just some minor improvements. The vm template change was required for my RHEL8 CSB build.

### CODE REVIEW

File: inventory/group_vars/all.yaml [hunk 1]
```diff
@@ -13,19 +14,19 @@ fcos_image_name: fedora-coreos-31.20200118.3.0-qemu.x86_64.qcow2
...
-  username: raffa
-  password: raffa
+  username: "raffa"
+  password: "raffa"
   pull_secret_file: pull_secret.json
   sigstore: https://mirror.openshift.com/pub/openshift-v4/signatures/
   tang_secret: PLjNyRdGw03zlRoGjQYMahSZGu9
 
 cluster:
-  version: 4.3.3
```
Comment (raffaelespazzoli) [inline inventory/group_vars/all.yaml:28]: why this change?

File: roles/create-vms/templates/vm-template.xml.j2 [hunk 2]
```diff
@@ -3,7 +3,7 @@
   <memory unit='MB'>{{ item.memory }}</memory>
   <vcpu>{{ item.cpus }}</vcpu>  
   <os>
-    <type arch="x86_64" machine="pc-i440fx-4.1">hvm</type>
```
Comment (raffaelespazzoli) [inline roles/create-vms/templates/vm-template.xml.j2:6]: this might be fine as I copied the setting from somewhere, but can you explain what your change does.

**Review by raffaelespazzoli — COMMENTED** [review]
please see my comments.

File: inventory/group_vars/all.yaml (hunk 1 above)
Comment (funkytaco) [reply inventory/group_vars/all.yaml:28]: Accidental left over config. I was thinking maybe the verison could be moved to the .env file and be set there? e.g. OC_CLUSTER_VERSION?

File: roles/create-vms/templates/vm-template.xml.j2 (hunk 2 above)
Comment (funkytaco) [reply roles/create-vms/templates/vm-template.xml.j2:6]: Qemu docs say it is best to not specify machine version, plus it doesn't work on RHEL8 CSB build.
